commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java

```diff
@@ -82,11 +83,21 @@ public class Web3SignerBaseCommand implements Config, Runnable {
 
   @Option(
       names = {"--key-store-path"},
-      description = "The path to a directory storing toml files defining available keys",
+      description = "The path to a directory storing yaml files defining available keys",
       paramLabel = DefaultCommandValues.MANDATORY_PATH_FORMAT_HELP,
       arity = "1")
   private Path keyStorePath = Path.of("./");
 
+  @Option(
+      names = {"--key-store-config-file-size"},
+      description =
+          "The key store configuration file size in bytes. Useful when loading large number of "
+              + "configurations from same yaml file. Defaults to (${DEFAULT-VALUE}) 100MB",
```

Comment:
The second sentence would be clearer as "Useful when loading a large number of configurations from the same yaml file"